website/index.html

```diff
@@ -101,14 +101,14 @@ <h1 id="deno"><a href="#deno">#</a>Deno</h1>
 
       <h2 id="install"><a href="#install">#</a>Install</h2>
 
-      <p id="install-shell"><a href="#install-shell">#</a>With Shell</p>
+      <p id="install-shell">With Shell</p>
       <pre>curl -fL https://deno.land/x/install/install.sh | sh</pre>
-      <p id="install-powershell"><a href="#install-powershell">#</a>With PowerShell</p>
+      <p id="install-powershell">With PowerShell</p>
       <pre>iex (iwr https://deno.land/x/install/install.ps1)</pre>
 
       <h2 id="mini-tutorial"><a href="#mini-tutorial">#</a>Mini-tutorial</h2>
 
-      <p id="try"><a href="#try">#</a>Try a Deno program. This one serves a local directory in HTTP.</p>
```

Comment:
I think this is useful if we want to link usage example

Reply:
There's an anchor directly above this in the title of the section...

Reply:
Okay